src/main/java/de/kekru/struktogrammeditor/control/GlobalSettings.java

```diff
@@ -67,25 +69,27 @@
 		readBuildInfoFile();
 	}
 
-	public static void init(){
-		for(int i=0; i < EinstellungsDialog.anzahlStruktogrammElemente; i++){
-			elementBeschriftungenZumEinfuegenInDasStruktogramm[i] = EinstellungsDialog.standardWerte[i];
+	public static void init() {
+		for (int i = 0; i < EinstellungsDialog.anzahlStruktogrammElemente; i++){
+			elementBeschriftungenZumEinfuegenInDasStruktogramm.add(EinstellungsDialog.standardWerte[i]);
 		}
 		
 		loadSettings();
 	}
 	
-	private static void readBuildInfoFile(){
-
+	private static void readBuildInfoFile () {
 		try {
-	
 			Properties pr = new Properties();
-
 			InputStream in = null;
 			try {
-				in = new BufferedInputStream(GlobalSettings.class.getResourceAsStream(BUILDINFO_FILE));
-				pr.load(in);
-				in.close();
+				if (GlobalSettings.class.getResourceAsStream(BUILDINFO_FILE) == null) {
+					System.err.println("Error no Buildinfo file could be found");
+				} else {
+					in = new BufferedInputStream(GlobalSettings.class.getResourceAsStream(BUILDINFO_FILE));
+					pr.load(in);
+					in.close();
+					return;

```

Comment:
Das return muss raus, sonst wird er nie weiter unten die Werte aus "pr" auslesen.  
Zum testen macht du am besten einmal ein `mvn clean package` und kopierst dir dann die `target/generated/build-metadata/build.properties` nach `src/main/resources/build.properties`  

Aber nicht in Git einchecken